octobot_commons/singleton/singleton_class.py

```diff
@@ -17,3 +17,14 @@ def instance(cls, *args, **kwargs):
         if cls not in cls._instances:
             cls._instances[cls] = cls(*args, **kwargs)
         return cls._instances[cls]
+    
+    @classmethod
+    def get_instance_if_exists(cls):
+        """
+        Return the instance if it exist
+        Return the class instance if it exist
+        :return: the class only instance if it exist otherwise None
+        """
+        if cls in cls._instances:
+            return cls._instances[cls]
+        return None
```

Comment:
Sorry i just realized that it should be more pythonic to do : 
```suggestion
        try:
            return cls._instances[cls]
        except KeyError:
            return None
```

Reply:
👍